std/StandardToken.sol

```diff
@@ -1,5 +1,3 @@
-pragma solidity ^0.4.0;
```

Comment:
If we decide to apply changes to these files I'd upgrade the pragma the same way the docs are (version + comment).